examples/macropad_mp3/macropad_mp3.py

```diff
@@ -0,0 +1,32 @@
+# SPDX-FileCopyrightText: Copyright (c) 2021 Kattni Rembor for Adafruit Industries
```

Comment:
should this file be examples/macropad_mp3/code.py? I am not sure about the new rules.

Reply:
This works! Updating!

Reply:
For library code, Bundlefly automatically changes the file to code.py so that the files can continue to stay within library standards and Adabot remains happy.